packages/account/integration-tests/personal-details/menu.spec.tsx

```diff
@@ -1,12 +1,15 @@
 import { test, expect } from '@playwright/test';
-import { mock_residents_list, mock_states_list, mock_general, mock_loggedIn, setupMocks } from '@deriv/integration';
+import { DEFAULT_ACCOUNTS, mock_general, mock_loggedIn, setupMocks } from '@deriv/integration';
 
 test.describe('Personal Details - Menu', () => {
     test('the navigation menu works', async ({ page, baseURL }) => {
         await setupMocks({
             baseURL,
+            state: {
+                accounts: DEFAULT_ACCOUNTS,
+            },
             page,
-            mocks: [mock_general, mock_loggedIn, mock_residents_list, mock_states_list],
+            mocks: [mock_general, mock_loggedIn],
```

Comment:
@markwylde-deriv Can we rename these functions to camelCase since this PR is about refactoring :) ?

Reply:
Was hoping to do this in a separate PR, but at least it's done now 😄